src/main/resources/db/migration/V005__Process_events.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE process_events (
+  id BIGSERIAL PRIMARY KEY,
+  container VARCHAR(50) NOT NULL,
+  zipFileName VARCHAR(255) NOT NULL,
+  createdAt TIMESTAMP NOT NULL,
+  envelope_id VARCHAR(50) NULL,
```

Comment:
Do we need envelope_id in process_events table ? Say we fail to process zip(generic failure) or we are creating event for AV failed envelope then when we create a record in process_events table we won't have envelope id. It will look like an orphan record ?

If we create a record in envelope table for above scenarios then it is fine to have envelope id as you can find them in both tables.

Reply:
Is there a scenario where we will be retrieving records from event table based on envelope id ?